### PR TITLE
Cleanup and add all the log properties in systems service

### PR DESCRIPTION
**Pull request by arahimanshaik**
- Added all logs properties in the systems service
- Added debug logs

### CODE REVIEW

**Review by AngithCJ — APPROVED** [review]

**Review by chauberahul1993 — APPROVED** [review]

File: svc-systems/chassis/create.go [hunk 1]
```diff
@@ -186,9 +180,9 @@ type linksManagedByCollection struct {
 	}
 }
 
-//{
-//	"Name" : "name"
-//}
+//	{
```
Comment (hemkapoor) [inline svc-systems/chassis/create.go:183]: do we need this?

File: svc-systems/chassis/fabricresource.go [hunk 2]
```diff
@@ -34,18 +36,23 @@ var (
...
-		l.Log.Warn("while trying to collect fabric managers details from DB, got " + err.Error())
+		l.LogWithFields(ctx).Warn("while trying to collect fabric managers details from DB, got " + err.Error())
 		return common.GeneralError(http.StatusNotFound, response.ResourceNotFound, "", []interface{}{"Chassis", rID}, nil)
 	}
 
+	threadID := 1
 	for _, manager := range managers {
-		go f.getResource(manager, rID, ch)
+		ctxt := context.WithValue(ctx, common.ThreadName, common.CollectChassisResource)
+		ctx = context.WithValue(ctxt, common.ThreadID, strconv.Itoa(threadID))
```
Comment (hemkapoor) [inline svc-systems/chassis/fabricresource.go]: it should be ctxt ..no? and we should pass ctxt in getResource function

File: svc-systems/chassis/updatefabricresource.go [hunk 3]
```diff
@@ -35,18 +37,22 @@ var (
...
-		l.Log.Warn("while trying to collect fabric managers details from DB, got " + err.Error())
+		l.LogWithFields(ctx).Warn("while trying to collect fabric managers details from DB, got " + err.Error())
 		return common.GeneralError(http.StatusNotFound, response.ResourceNotFound, "", []interface{}{"Chassis", url}, nil)
 	}
-
+	var threadID int = 1
 	for _, manager := range managers {
-		go f.updateResource(manager, url, body, ch)
+		ctxt := context.WithValue(ctx, common.ThreadName, common.UpdateChassisResource)
+		ctx = context.WithValue(ctxt, common.ThreadID, strconv.Itoa(threadID))
```
Comment (hemkapoor) [inline svc-systems/chassis/updatefabricresource.go]: here as well..it should be ctxt

File: svc-systems/chassis/create.go (hunk 1 above)
Comment (arahimanshaik) [reply svc-systems/chassis/create.go:183]: removed it

File: svc-systems/chassis/fabricresource.go (hunk 2 above)
Comment (arahimanshaik) [reply svc-systems/chassis/fabricresource.go]: fixed

**Review by hemkapoor — APPROVED** [review]

**Review by kavyamani596 — APPROVED** [review]

**Review by amar-shalgar — APPROVED** [review]

**Pull request merged**